test/tskiplist.c

```diff
@@ -1555,7 +1555,7 @@ test_skiplist_term(void)
 **
 ****************************************************************/
 void
-test_skiplist(void)
```

Comment:
`test_skiplist()` has a initialization routine (`test_skiplist_init()`) and a termination routine (`test_skiplist_term()`) that handle some global buffers. Are the test-framework level setup/cleanup functions just for HDF5-library level setup and cleanup?

Reply:
I hadn't looked into the content of any of the tests for possible changes. In this case, the test's setup and cleanup is performed just once before and after all the tests, where the test setup and cleanup callbacks are used on a per-test basis. Though it could make sense to allow specifying pre- and post-tests callbacks in `TestInit`, for example.